### DISCUSSION TITLE
Setup Build output folder.

### DISCUSSION THREAD

**Discussion by Ghotic190** [stryker-mutator/stryker-net#2789] [Q&A]
Hello guys, I'm trying to setup Stryker on my solution but i need navigate stryker to my output folder \..\..\bin\x64\Debug\  is there some option to do this because stryker is trying to find binaries in bin\Debug\ of the test-project path.

**Comment by rouke-broersma** [comment]
This sounds like https://github.com/stryker-mutator/stryker-net/issues/2077
Unfortunately I don't know the fix for this issue
